runtime-services-modeling/deploying-services.adoc

```diff
@@ -919,20 +238,20 @@ The following tables list all the supported add-ons:
 |Messaging                | kogito-addons-quarkus-messaging                     |1.0.0
 |Events Decisions         | kogito-addons-quarkus-events-decisions              |1.2.0
 |Events Process           | kogito-addons-quarkus-events-process                |0.3.0
-|Explainability           | kogito-addons-quarkus-explainability                |0.15.0
-|Jobs Management          | kogito-addons-quarkus-jobs-management               |0.6.0
+// |Explainability           | kogito-addons-quarkus-explainability                |0.15.0
+// |Jobs Management          | kogito-addons-quarkus-jobs-management               |0.6.0
 |Knative Eventing         | kogito-addons-quarkus-knative-eventing              |1.10.0
 |Mail                     | kogito-addons-quarkus-mail                          |1.6.0
 |Elastic Monitoring       | kogito-addons-quarkus-monitoring-elastic            |1.2.0
 |Prometheus Monitoring    | kogito-addons-quarkus-monitoring-prometheus         |0.1.0
-|Persistence ISPN Health  | kogito-addons-quarkus-persistence-infinispan-health |0.8.0
-|Persistence Kafka        | kogito-addons-quarkus-persistence-kafka             |1.4.0
-|Process Management       | kogito-addons-quarkus-process-management            |0.3.0
-|Process SVG              | kogito-addons-quarkus-process-svg                   |1.0.0
-|Task Management          | kogito-addons-quarkus-task-management               |1.4.0
-|Task Notification        | kogito-addons-quarkus-task-notification             |1.6.0
+// |Persistence ISPN Health  | kogito-addons-quarkus-persistence-infinispan-health |0.8.0
+// |Persistence Kafka        | kogito-addons-quarkus-persistence-kafka             |1.4.0
+// |Process Management       | kogito-addons-quarkus-process-management            |0.3.0
+// |Process SVG              | kogito-addons-quarkus-process-svg                   |1.0.0
+// |Task Management          | kogito-addons-quarkus-task-management               |1.4.0
+// |Task Notification        | kogito-addons-quarkus-task-notification             |1.6.0
 |Tracing Decision         | kogito-addons-quarkus-tracing-decision              |0.11.0
-|Events MongoDB           | kogito-addons-quarkus-events-mongodb                |1.11.0
+// |Events MongoDB           | kogito-addons-quarkus-events-mongodb                |1.11.0
 |Source Files             | kogito-addons-quarkus-source-files                  |1.20.0
 |===
```

Comment:
At some point, I'd like to see the table on rows instead of columns, but that isn't important for this PR.

|===
| Header info | Column two

| 1r1c
|1r2c

|2r1c
|2r2c

|===